Gemfile

```diff
@@ -10,8 +10,8 @@ group :test, :development do
   gem 'rspec', '~> 3.1'
 
   unless ENV['NO_ACTIVERECORD']
-    gem 'activerecord', '>= 3.2.3', '< 4.3.0'
-    gem 'activerecord-oracle_enhanced-adapter', '>= 1.4.1', '< 1.7.0'
+    gem 'activerecord', '>= 3.2.3', '< 5.1.0'
```

Comment:
Looks like active record 5 doesn't support older versions of Ruby (it needs >= 2.2.2). We should somehow limit Rails to v4 for the old versions of Ruby. (At least for the purpose of Travis CI builds).